package.json

```
@@ -135,6 +139,15 @@
     "url": "https://github.com/plotly/react-chart-editor.git"
   },
   "browserslist": [
-    "ie 11"
-  ]
+    "defaults",
+    "not IE 11"
```

```suggestion
    "last 8 years and not dead"
```
I'm not quite sure what `"defaults"` means, but as I recall it was a little more aggressive about dropping older browsers than we were comfortable with in some contexts. The above is what we switched to in eg [dash-renderer](https://github.com/plotly/dash/blob/6eaf2e17c25f7ca1847c41aafeb18e87c586cb9f/dash/dash-renderer/package.json#L90) when we dropped IE support.